Python/Product/PythonTools/PythonTools/Debugger/DebugPyVersionHelper.cs

```diff
@@ -25,81 +25,81 @@
 using Newtonsoft.Json;
 
 namespace Microsoft.PythonTools.Debugger {
-    internal class PtvsdVersionArguments {
+    internal class DebugPyVersionArguments {
     }
 
-    internal class PtvsdVersionResponse : ResponseBody {
-        [JsonProperty("ptvsd")]
-        public PtvsdDebuggerVersion Debugger { get; set; }
+    internal class DebugPyVersionResponse : ResponseBody {
+        [JsonProperty("debugpy")]
+        public DebugPyDebuggerVersion Debugger { get; set; }
 
         [JsonProperty("python")]
         public PythonVersionInfo Python { get; set; }
 
         [JsonProperty("platform")]
-        public PtvsdPlatfromInfo Platfrom { get; set; }
+        public DebugPyPlatfromInfo Platfrom { get; set; }
 
         [JsonProperty("process")]
-        public PtvsdProcessInfo Process { get; set; }
+        public DebugPyProcessInfo Process { get; set; }
 
     }
 
-    internal class PtvsdVersionRequest : DebugRequestWithResponse<PtvsdVersionArguments, PtvsdVersionResponse> {
-        public PtvsdVersionRequest(): base("ptvsd_systemInfo") {
+    internal class DebugPyVersionRequest : DebugRequestWithResponse<DebugPyVersionArguments, DebugPyVersionResponse> {
+        public DebugPyVersionRequest(): base("debugpySystemInfo") {
         }
     }
 
-    internal class PtvsdVersionHelper {
-        public static void VerifyPtvsdVersion(PtvsdVersionArguments args, PtvsdVersionResponse response) {
+    internal class DebugPyVersionHelper {
+        public static void VerifyDebugPyVersion(DebugPyVersionArguments args, DebugPyVersionResponse response) {
             if (PackageVersion.TryParse(response.Debugger.Version, out PackageVersion runningVersion)) {
-                var bundledPtvsdVersion = PackageVersion.Parse(PtvsdVersion.Version);
-                if (runningVersion.CompareTo(bundledPtvsdVersion) < 0) {
-                    ShowPtvsdMessage(
+                var bundledDebugPyVersion = PackageVersion.Parse(DebugPyVersion.Version);
+                if (runningVersion.CompareTo(bundledDebugPyVersion) < 0) {
+                    ShowDebuggingErrorMessage(
                         Strings.InstalledPtvsdOutdatedTitle,
-                        Strings.InstalledPtvsdOutdatedMessage.FormatUI(response.Debugger.Version, PtvsdVersion.Version),
+                        Strings.InstalledPtvsdOutdatedMessage.FormatUI(response.Debugger.Version, DebugPyVersion.Version),
```

Comment:
this one and the next will need to say debugpy in the message rather than ptvsd.